stock_move_line_auto_fill/models/stock_picking_type.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = 'stock.picking.type'
+
+    auto_fill_operation = fields.Boolean(
+        string="Auto fill operations",
+    )
```

Comment:
help="To auto fill done quantity in picking document.\n- If checked, auto fill done quantity automatically\n- If unchecked, show button AutoFill for user to fill done quantity manually."

Reply:
Changed